src/map/entities/charentity.h

```diff
@@ -542,8 +541,7 @@ class CCharEntity : public CBattleEntity
 
     CItemEquipment* getEquip(SLOTTYPE slot);
 
-    // TODO: Don't use raw ptrs for this, but don't duplicate whole packets with unique_ptr either.
-    CBasicPacket* PendingPositionPacket = nullptr;
+    std::deque<std::unique_ptr<CBasicPacket>> PacketList; // The list of packets to be sent to the character during the next network cycle
```

Comment:
instead of making this public, can you keep it private and access it with: `auto getPacketList() -> std::deque<std::unique_ptr<CBasicPacket>>&`

Reply:
Good call, made it const too.